css/index.css

```diff
@@ -1,3 +1,85 @@
+*{
+    box-sizing: border-box;
+}
+
+.whole {
+    padding: 10px 3%;
+    margin: auto; 
+}
+
+header {
+    display: flex;
+    justify-content: space-between;
+    align-items: center;
+    padding:10px 20px;
+    position: fixed; 
+    top: 0;
+    width: 100%;
+    transition: 0.6s;
+    margin-bottom: 5px;
+    float: right;
+    background-color: lightgray;
+
+}
+
+header.sticky {
+    padding: 5px 10px;
+    background:rgb(3, 68, 122);
```

Comment:
small nitpick; but generally it's good practice to use the same way of writing colours throughout your codebase. So if you use `rgb(..)` , use that for all your colours and backgrounds for example. Instead of switching between hex (`#000`) , `rgb` and the string names (`bisque`).